cmd/aries-js-worker/src/aries.js

```diff
@@ -14,26 +14,70 @@ export const Aries = new function() {
     // TODO synchronize access on this map?
     this._pending = new Map()
 
-    /**
-     * Test methods.
-     * TODO - remove. Used for testing.
-     * @type {{_echo: (function(*=): Promise<String>)}}
-     * @private
-     */
-    this._test = {
```

Comment:
can you leave these in for now so we can easily test js-wasm integration?

Reply:
done